v2/src/runner.py

```diff
@@ -72,9 +72,28 @@ def _create_command(spec: _RunnerSpec) -> str:
     )
 
 
-def _create_commands(spec: _RetrySpec) -> Sequence[str]:
-    commands = []
-    previous_output = None
+def _create_merge_command(
+    *,
+    python_executable: pathlib.Path,
+    variants_outputs: Iterable[pathlib.Path],
+    final_output: pathlib.Path,
+) -> str:
+    return (
+        f"{python_executable} -m robot.rebot --output={final_output} --report=NONE --log=NONE "
+        f"{' '.join(str(variant_output) for variant_output in variants_outputs)}"
+    )
+
+
+@dataclasses.dataclass(frozen=True)
+class _RetryCommands:
```

Comment:
I am not really happy with this name and thus open to suggestions

Reply:
It seems to me describe the content well enough. So, I would rethink the class but see my comment about the merge command.